### PR TITLE
Added WP CLI command for flushing caches.

### PR DESCRIPTION
**Pull request by richardkorthuis**

### CODE REVIEW

File: includes/cli/class-flush-command.php [hunk 1]
```diff
@@ -0,0 +1,78 @@
+<?php
```
Comment (acato-plugins) [inline includes/cli/class-flush-command.php:1]: :no_entry_sign: **Error**: Class file names should be based on the class name with &quot;class-&quot; prepended. Expected class-flush-command.php, but found phpcs-scan-3yuYoP.php (*WordPress.Files.FileName.InvalidClassFileName*).

Comment (acato-plugins) [inline includes/cli/class-flush-command.php:1]: :no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-3yuyop.php, but found phpcs-scan-3yuYoP.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).

**Review by acato-plugins — CHANGES_REQUESTED** [review]
**phpcs** scanning turned up:

:no_entry_sign: 2 errors 

***

**hashes-api**-scanning skipped

**Comment by acato-plugins** [comment]
GitHub API communication error. Please contact a human. (commit-ID: 567f4c424018f780868e46de1bc28e78983b15f4).

**Review by yoeridekker — APPROVED** [review]
Nice!

**Pull request merged**